docs/community/contributor-guide.md

```diff
@@ -0,0 +1,539 @@
+# How to contribute to `physiopy`
```

Comment:
did you build the site locally ? from what I expect you wouldn't get the automatic table of contents if links are not embedded into titles...

Like this : `[#title](#link)`

Reply:
I did build it locally! And actually, this is the correct way to do it 😉  The links are automatically generated by mkdocs.